lib/fountain/api/applicants.rb

```diff
@@ -137,6 +137,26 @@ def self.advance_applicant(applicant_id, advance_options = {})
         true
       end
 
+      #
+      # Advance multiple Applicants
+      # @param [String] applicant_id IDs of the Fountain applicants
+      # @param [String] Destination stage's ID
+      # @param [Hash] advance_options A hash of options to advance applicant
+      #                 skip_automated_actions - `true` if you want to skip automated
+      #                                          actions when advancing the applicant
+      def self.advance_applicants(applicant_ids, stage_id, advance_options = {})
+        response = request(
+          "/v2/applicants/advance?#{stage_id}",
+          method: :post,
+          body: Util.slice_hash(
+            advance_options,
+            :skip_automated_actions
```

Comment:
the API also supports an optional `funnel_id` param. Should that be included here?